src/ar/mangaswat/src/eu/kanade/tachiyomi/extension/ar/mangaswat/MangaSwat.kt

```diff
@@ -95,105 +105,228 @@ class MangaSwat :
     }
 
     override fun latestUpdatesRequest(page: Int): Request {
-        val xhrHeaders = headersBuilder()
-            .add("Content-Type", "application/x-www-form-urlencoded; charset=UTF-8")
-            .add("X-Requested-With", "XMLHttpRequest")
+        val apiHeaders = headersBuilder()
+            .add("Accept", "application/json, text/plain, */*")
+            .add("Origin", baseUrl)
+            .add("Referer", "$baseUrl/")
             .build()
+        return GET("$apiBaseUrl/series/?status=79&page=$page", apiHeaders)
+    }
 
-        val formBody = FormBody.Builder()
-            .add("action", "more_manga_home")
-            .add("paged", (page - 1).toString())
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        val data = json.decodeFromString<LatestUpdatesResponse>(response.body.string())
```

Comment:
Use `parseAs` from `keiyoushi.utils`
```suggestion
        val data = response.parseAs<LatestUpdatesResponse>()
```